docs/template_plugin/src/CMakeLists.txt

```diff
@@ -27,18 +27,14 @@ target_include_directories(${TARGET_NAME} PRIVATE
 target_link_libraries(${TARGET_NAME} PRIVATE
     IE::inference_engine
     IE::inference_engine_transformations
+    interpreter_backend
+    ngraph::reference
```

Comment:
why `ngraph::reference` are needed?

Reply:
We use reference headers in the template plugin